alpine/Dockerfile

```diff
@@ -93,6 +93,9 @@ VOLUME ${OPENHAB_CONF} ${OPENHAB_USERDATA} ${OPENHAB_HOME}/addons
 # Expose HTTP, HTTPS, Console and LSP ports
 EXPOSE 8080 8443 8101 5007
 
+# Set healthcheck
+HEALTHCHECK --interval=5m --timeout=5s --retries=3 CMD curl -f http://localhost:8080/ || exit 1
```

Comment:
Would it be possible to use the `OPENHAB_HTTP_PORT` environment variable here instead of the `8080` default? This health check would fail whenever a container is created with a custom value.

Reply:
You are absolutely right. I should have thought of that. I'll change it and test it